### PR TITLE
Tests updated

### PR DESCRIPTION
**Pull request by lmikelionis**

### CODE REVIEW

**Comment by mindaugasbarysas** [comment]
Strict standard errors.

File: Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php [hunk 1]
```diff
@@ -239,4 +240,86 @@ public function testIvalidateDefaultUrls()
...
+        }
+
+        if ($property !== null) {
+            $property = $class->getProperty($property);
+            $property->setAccessible(true);
+
+        }
+
+        $router = $this->getMock('Symfony\Component\Routing\RouterInterface');
+        $goodObj = new $className(
```
Comment (mindaugasbarysas) [inline Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php]: I'm not sure it's that good an object.

File: Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php [hunk 2]
```diff
@@ -239,4 +240,85 @@ public function testIvalidateDefaultUrls()
...
+            $method->setAccessible(true);
+        }
+
+        if ($property !== null) {
+            $property = $class->getProperty($property);
+            $property->setAccessible(true);
+        }
+
+        $router = $this->getMock('Symfony\Component\Routing\RouterInterface');
+        $goodObj = new $className(
```
Comment (mindaugasbarysas) [inline Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php]: What's a good object?

Comment (lmikelionis) [inline Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php]: Ojcet thats good for use :)

Comment (mindaugasbarysas) [inline Tests/Unit/UrlInvalidator/UrlInvalidatorServiceTest.php]: Why couldn't it be an `instance`?

**Pull request merged**